Source/StatisticsFunctions/arm_accumulate_q15.c

```diff
@@ -57,11 +57,11 @@
 void arm_accumulate_q15(
                         const q15_t * pSrc,
                         uint32_t blockSize,
-                        q15_t * pResult)
+                        q63_t * pResult)
 {
   uint32_t  blkCnt;           /* loop counters */
   q15x8_t  vecSrc;
-  q31_t     sum = 0L;
+  q63_t     sum = 0L;
```

Comment:
This is not consistent with the MVE instruction used : vaddvaq is using a int32.
So either you accumulate on int32 only. Or you need to internally expand the values to int32 and then use the MVE instruction to accumulate on int64.